### ISSUE TITLE
Request for chroot: openSUSE Leap 15.6

### ISSUE THREAD

**Issue by evgeni** [fedora-copr/copr#3371]
openSUSE Leap 15.6 is out since some time and it would be cool to have it available on Copr! :)

The config should be mostly `s/15\.5/15.6/` from the 15.5 one.

BTW, is there a reason not to use a "generic" Leap 15 chroot that uses the `:15` containers from openSUSE instead of the .y ones?

**Comment by FrostyX** [comment]
We will need to add 15.6 configs to more-core-configs first
https://github.com/rpm-software-management/mock/tree/main/mock-core-configs/etc/mock

> BTW, is there a reason not to use a "generic" Leap 15 chroot that uses the :15 containers from openSUSE instead of the .y ones?

That's a very good question. @Conan-Kudo you take care of the openSUSE configs in the Mock repo. What is your opinion on this?

**Comment by evgeni** [comment]
> > BTW, is there a reason not to use a "generic" Leap 15 chroot that uses the :15 containers from openSUSE instead of the .y ones?
> 
> That's a very good question. @Conan-Kudo you take care of the openSUSE configs in the Mock repo. What is your opinion on this?

Semi-answering myself:
While there is a "rolling" `leap:15` container, there is no `15` folder at https://download.opensuse.org/distribution/leap/, so while we can launch a container, we can't setup repos properly :cry:

**Comment by Conan-Kudo** [comment]
This is a question for @lkocman.

**Comment by praiskup** [comment]
Addded to copr.  Somewhat related https://github.com/rpm-software-management/mock/pull/1557

**Comment by Conan-Kudo** [comment]
> BTW, is there a reason not to use a "generic" Leap 15 chroot that uses the :15 containers from openSUSE instead of the .y ones?

I don't think that exists for `leap-dnf` containers. I didn't think it even existed for the regular leap ones...
